app/scripts/directives/annotationReport.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+angular.module('oncokbApp')
+    .directive('annotationReport', function(mainUtils, _) {
+        return {
+            templateUrl: 'views/annotationReport.html',
+            restrict: 'E',
+            scope: {
+                hugoSymbol: '=',
+                alteration: '=',
+                tumorType: '=',
+                data: '=',
+            },
+            link: function postLink(scope) {
+                function processLevel(levelOfEvidence) {
+                    var level = levelOfEvidence.replace('LEVEL_', '');
+                    return '<span class="oncokb level-' + level + '">Level ' + _.upperFirst(level) + '</span>';
```

Comment:
I didn't see any css called `level-*` in main.scss

Reply:
@victoria34 it's included in the index.html via cdn